samples/bluetooth/conn_time_sync/README.rst

```diff
@@ -0,0 +1,210 @@
+  Sent to conn index 0: c_anchor point (time=0, counter=0), c_trigger_time 4326732, value 0
+  Sent to conn index 0: c_anchor point (time=0, counter=0), c_trigger_time 4454845, value 1
+  Sent to conn index 0: c_anchor point (time=0, counter=0), c_trigger_time 4582927, value 0
+  Sent to conn index 0: c_anchor point (time=0, counter=0), c_trigger_time 4711101, value 1
+  Device found: FA:BB:79:57:D6:45 (random) (RSSI -42)
+  Connected: FA:BB:79:57:D6:45 (random)
+  Sent to conn index 0: c_anchor point (time=0, counter=0), c_trigger_time 4839336, value 0
+  Service discovery completed
+  Sent to conn index 0: c_anchor point (time=0, counter=0), c_trigger_time 4967601, value 1
+  Sent to conn index 1: c_anchor point (time=0, counter=0), c_trigger_time 4967601, value 1
+  Sent to conn index 0: c_anchor point (time=0, counter=0), c_trigger_time 5104076, value 0
+  Sent to conn index 1: c_anchor point (time=0, counter=0), c_trigger_time 5104076, value 0
+
+For the peripheral(s)::
+
+  Starting connection time sync sample.
+  Choose device role - type c (central) or p (peripheral):
+  Peripheral. Starting advertising
+  I: SoftDevice Controller build revision:
+  I: 43 77 a9 de 2d 55 e0 08 |Cw..-U..
+  I: 0a c7 0f e5 07 18 61 5b |......a[
+  I: 72 dd e6 08             |r...
+  I: HW Platform: Nordic Semiconductor (0x0002)
+  I: HW Variant: nRF52x (0x0002)
+  I: Firmware: Standard Bluetooth controller (0x00) Version 67.43383 Build 3763678686
+  I: Identity: FA:BB:79:57:D6:45 (random)
+  I: HCI: version 5.4 (0x0d) revision 0x1208, manufacturer 0x0059
+  I: LMP: version 5.4 (0x0d) subver 0x1208
+  Advertising started
+  Connected: E8:DC:8D:B3:47:69 (random)
+  Received: c_anchor point (time=0, counter=0), c_trigger_time 71124066, value 1
+  Received: c_anchor point (time=0, counter=0), c_trigger_time 71260388, value 0
+  Received: c_anchor point (time=0, counter=0), c_trigger_time 71396496, value 1
+  Received: c_anchor point (time=0, counter=0), c_trigger_time 71532788, value 0
+  Received: c_anchor point (time=0, counter=0), c_trigger_time 71669079, value 1
+  Received: c_anchor point (time=0, counter=0), c_trigger_time 71805371, value 0
+  Received: c_anchor point (time=0, counter=0), c_trigger_time 71941662, value 1
+  Received: c_anchor point (time=0, counter=0), c_trigger_time 72077984, value 0
+  Received: c_anchor point (time=0, counter=0), c_trigger_time 72214428, value 1
+  Received: c_anchor point (time=0, counter=0), c_trigger_time 72350720, value 0
+  Received: c_anchor point (time=0, counter=0), c_trigger_time 72486859, value 1
+  Received: c_anchor point (time=0, counter=0), c_trigger_time 72623303, value 0
+  Received: c_anchor point (time=0, counter=0), c_trigger_time 72759442, value 1
+  Received: c_anchor point (time=0, counter=0), c_trigger_time 72895733, value 0
+
+Dependencies
+*************
+
+This sample uses the following `sdk-nrfxlib`_ library:
```

Comment:
```suggestion
This sample uses the following `sdk-nrfxlib` library:
```

Reply:
This is the way to use links with sphinx, see output on https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-16550/nrf/samples/bluetooth/conn_time_sync/README.html